### ISSUE TITLE
YAML Front-Matter support would be awesome

### ISSUE THREAD

**Issue by teeramusic** [htmlhint/HTMLHint#204]
Currently unsupported as far as I can tell. But that would be quite awesome :)

**Comment by GarthDB** [comment]
Yes please, it's enough to make me ditch htmlhint at this point.

**Comment by mattccrampton** [comment]
is this still a pending feature?

**Comment by groenroos** [comment]
This issue should be re-opened.

Since HTMLHint doesn't currently ignore frontmatter, having indented frontmatter in a file clashes with the `"space-tab-mixed-disabled": "tab"` rule. YAML cannot be indented with tabs, so the linter throws up a lot of undesirable and unfixable errors.

**Comment by groenroos** [comment]
@coliff Could this issue be re-opened and kept unstale, please? 🙏
